src/topp/FileFilter.cpp

```diff
@@ -408,7 +412,7 @@ class TOPPFileFilter :
     registerTOPPSubsection_("f_and_c", "Feature & Consensus data options");
     registerStringOption_("f_and_c:charge", "[min]:[max]", ":", "Charge range to extract", false);
     registerStringOption_("f_and_c:size", "[min]:[max]", ":", "Size range to extract", false);
-    registerStringList_("f_and_c:remove_meta", "<name> 'lt|eq|gt' <value>", StringList(), "Expects a 3-tuple (=3 entries in the list), i.e. <name> 'lt|eq|gt' <value>; the first is the name of meta value, followed by the comparison operator (equal, less or greater) and the value to compare to. All comparisons are done after converting the given value to the corresponding data value type of the meta value (for lists, this simply compares length, not content!)!", false);
+    registerStringList_("f_and_c:remove_meta", "<name> 'lt|eq|gt' <value>", StringList(), "WARNING: THIS FLAG IS DEPRECATED. USE remove_meta instead.   Expects a 3-tuple (=3 entries in the list), i.e. <name> 'lt|eq|gt' <value>; the first is the name of meta value, followed by the comparison operator (equal, less or greater) and the value to compare to. All comparisons are done after converting the given value to the corresponding data value type of the meta value (for lists, this simply compares length, not content!)!", false);
```

Comment:
why move the flag out of `f_and_c`? It effectively only works for these data types

Reply:
there is extant code applying it to MzML files as well. I can either move it out of f_and_c or remove the code.

Reply:
I dont see how `remove_meta` currently applies to mzML files, did you add that code somewhere?